reference-architecture/osp-cli/ch4.8.4_enable_docker_nodes.sh

```diff
@@ -0,0 +1,5 @@
+for H in $NODES
+do
+  ssh $H sudo systemctl enable docker
```

Comment:
Enable could be left for the installer but since it was already installed and configured, not enabling it would raise questions.